src/js/view/controls/components/button.js

```diff
@@ -21,7 +21,7 @@ export default function (icon, apiAction, ariaText, svgIcons) {
     }
 
     // prevent button from having a border when focused through click
-    addFocusHandler(element);
+    flagNoFocus(element);
```

Comment:
Which buttons don't have an `apiAction`?  If `UI` handles focus correction then we should always use it, and not use `addFocusHandler` and/or `flagNoFocus`.